GenericsHomework.Tests/NodeTests.cs

```diff
@@ -0,0 +1,358 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace GenericsHomework.Tests
+{
+
+    [TestClass]
+    public class NodeTests
+    {
+
+        public string ToString(int[] array) => "{" + string.Join(", ", array) + "}";
+        public string ToString(ArrayList list) => "{" + string.Join(", ", list.ToArray()) + "}";
+        public string ToString<T>(List<T> list) => "{" + string.Join(", ", list.ToArray()) + "}";

```

Comment:
These can all be static.
```suggestion
        public static string ToString(int[] array) => "{" + string.Join(", ", array) + "}";
        public static string ToString(ArrayList list) => "{" + string.Join(", ", list.ToArray()) + "}";
        public static string ToString<T>(List<T> list) => "{" + string.Join(", ", list.ToArray()) + "}";
```